### PR TITLE
Added basic navigation bar.

### PR DESCRIPTION
**Pull request by impguard**

### CODE REVIEW

**Comment by whsieh** [comment]
It's failing cucumber because I implemented one of the steps that were previously pending on another feature to use in my own feature, but this causes the other feature to fail.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
+include AccessTokenHelper
```
Comment (impguard) [inline app/controllers/application_controller.rb]: ?

File: config/routes.rb [hunk 2]
```diff
@@ -16,6 +16,9 @@
     post "/api/v1/user/:id/profile" => "users#post_profile_info_by_id"
     get "/api/v1/arenas/:user_id" => "arenas#get_by_user_id"
 
+    # Public routes
+    get "/home" => "application#homepage"
```
Comment (impguard) [inline config/routes.rb:20]: weird. probably remove when we add the global portal system.

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,2 +1,7 @@
...
+
 class ApplicationController < ActionController::Base
+    def homepage
+        render :file => "public/homePage.html"
```
Comment (impguard) [inline app/controllers/application_controller.rb:4]: same here!
tag it with #TODO

**Comment by leohtkam** [comment]
+1

**Comment by impguard** [comment]
+1

**Pull request merged**